src/app/src/androidTest/java/br/com/italoromeiro/arctouchtest/robots/SearchRobot.java

```diff
@@ -0,0 +1,24 @@
+package br.com.italoromeiro.arctouchtest.robots;
+
+import br.com.italoromeiro.arctouchtest.R;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+/**
+ * Created by italo on 13/08/16.
+ */
+public class SearchRobot {
+    public SearchRobot where(String nameOfStreet) {
```

Comment:
What do you think about the name of this method? This makes the "robot" type a text into the search input and hide the keyboard, right? I'm not sure `where` makes clear that those actions will happen.

So I started thinking we could call it something like `typeIntoSearchInput` or something like that, but then I realised that every time you call `where()`, the `search()` method is called right after it (in the `ListTest` class). So maybe instead of having the two methods, we could just have one (maybe called `search()` like the one we already have) that does all the actions: type the text, hide the keyboard, and trigger the search. What do you think about that?